crates/parcel/src/plugins.rs

```diff
@@ -175,14 +182,36 @@ impl<'a> Plugins<'a> {
       };
     }
 
-    Ok(transformers)
+    Ok(TransformerPipeline {
+      transformers,
+      hash: hasher.finish(),
+    })
   }
 
   pub fn validators(&self, _path: &Path) -> Result<Vec<Box<dyn ValidatorPlugin>>, anyhow::Error> {
     todo!()
   }
 }
 
+pub struct TransformerPipeline {
+  pub transformers: Vec<Box<dyn TransformerPlugin>>,
```

Comment:
Still think this could just be a list of plugin names from the config that could be "loaded" on demand as they are run.